cmd/rm.go

```diff
@@ -24,44 +24,45 @@ import (
 )
 
 func rm(cmd *cobra.Command, args []string) error {
-	if len(args) != 1 {
-		return errors.New("`rm` requires a `file` argument")
+	if len(args) < 1 {
+		return errors.New("rm: missing operand")
 	}
 
-	path, err := validatePath(args[0])
-	if err != nil {
-		return err
-	}
-
-	dbx := files.New(config)
-	pathMetaData, err := getFileMetadata(dbx, path)
-	if err != nil {
-		return err
-	}
+	for i := range args {
+		path, err := validatePath(args[i])
```

Comment:
Sure, If there are multiple arguments and one doesn't exist, should the others still be removed?